be/src/olap/rowset/beta_rowset_reader.cpp

```diff
@@ -232,6 +232,8 @@ Status BetaRowsetReader::get_segment_iterators(RowsetReaderContext* read_context
         _read_options.io_ctx.query_id = &_read_context->runtime_state->query_id();
         _read_options.io_ctx.read_file_cache =
                 _read_context->runtime_state->query_options().enable_file_cache;
+        _read_options.io_ctx.is_disposable =
+                _read_context->runtime_state->query_options().disable_file_cache;
```

Comment:
reuse `_read_context->runtime_state->query_options().enable_file_cache`
because `disable_file_cache` is deprecated.

BTW remove `disable_file_cache` by renaming it to `DEPRECATED_disable_file_cache`

Reply:
目前disable_file_cache这个名字在社区还不能一下子废除，在导入路径是否写cache的时候会用到